dotlottie-rs/src/fms/dolottie_manager.rs

```diff
@@ -186,7 +183,9 @@ impl DotLottieManager {
 
             Ok(cloned_animation)
         } else {
-            let animation = crate::get_animation(&self.zip_data, animation_id);
+            let version = if is_v2(&self.manifest) { 2 } else { 1 };
```

Comment:
@theashraf This will need refactoring when we introduce more versions than v2, and it also hard codes it to default to 1, can we not change `is_v2 ` to a method that returns the version as a u8?

Reply:
@samuelOsborne resolved in https://github.com/LottieFiles/dotlottie-rs/pull/254/commits/d23b83a59f605db9b08c649f40f9b95f055d1ced